docs/HelpWantedProcess.md

```diff
@@ -0,0 +1,13 @@
+This document describes the process that the team uses for identifying candidate issues for community members to tackle.
+
+## Process
+
+This process (we'll refer to it as `help-wanted` process going forward) triggers from the moment, when the engineering team decides that a particular issue may be a good fit for a community member to tackle. Below are the stages that issue will go through, as part of this process:
+1. The triage team will **assign the `help-candidate` label** to the issue, as well as **assign an engineer to the issue**.
```

Comment:
Is there any process for community triagers to assign the help-candidate label?

Reply:
Good question. In my mind, you guys are trusted enough to label issues, which also includes this label.
I think it will even be possible for you to both nominate such candidate issues as well as actually add the summary comment and then ask someone from the team to review / approve it before adding the "help-wanted" label.